### PR TITLE
Sdk-1902:Doc scan watchlist screening check

### PR DESCRIPTION
**Pull request by LuborRod**
## Added

* Support for WATCHLIST_SCREENING check in Doc Scan

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
             <directory suffix=".php">src</directory>
             <exclude>
                 <directory>src/Protobuf</directory>
+                <directory>src/DocScan/Session/Retrieve/Contracts</directory>
```
Comment (echarrod) [inline phpunit.xml:21]: Generally we take the approach of only excluding generated files, so can we remove this line? 
```suggestion
```

**Review by echarrod — CHANGES_REQUESTED** [review]
Just a small PHPUnit exclusion change, think it looks good otherwise :+1:

File: phpunit.xml (hunk 1 above)
Comment (LuborRod) [reply phpunit.xml]: Yes

**Review by echarrod — APPROVED** [review]
:+1:

**Comment by echarrod** [comment]
Merging with failing quality gate as coverage will be added in a separate PR

**Pull request merged**